subxt/src/tx/tx_progress.rs

```diff
@@ -108,6 +111,9 @@ where
                 TxStatus::FinalityTimeout(_) => {
                     return Err(TransactionError::FinalitySubscriptionTimeout.into())
```

Comment:
Now we have these other errors, perhaps this variant should be renamed to `FinalityTimeout` to be consistent.